src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -87,6 +87,9 @@ public static LSP.TextDocumentPositionParams PositionToTextDocumentPositionParam
         public static LSP.TextDocumentIdentifier DocumentToTextDocumentIdentifier(Document document)
             => new LSP.TextDocumentIdentifier() { Uri = document.GetURI() };
 
+        internal static LSP.TextDocumentIdentifier? TextDocumentItemToTextDocumentIdentifier(LSP.TextDocumentItem textDocument)
```

Comment:
I cntrl+F for this, but didn't see any usages, is it needed?

Reply:
Nope! Good catch. I swear it was once :P